packages/fx-core/src/component/generator/sample.ts

```diff
@@ -0,0 +1,49 @@
+import sampleConfig from "./samples-config.json";
+
+export interface SampleInfo {
+  id: string;
+  title: string;
+  shortDescription: string;
+  fullDescription: string;
+  tags: string[];
+  time: string;
+  configuration: string;
+  link?: string;
+  suggested: boolean;
+  relativePath?: string;
+}
+
+export interface SampleCollection {
```

Comment:
Just curious why we need the interface of SampleCollection, it looks a little duplicated with SampleInfo[]